includes/orders/classes/class-order.php

```diff
@@ -280,6 +280,7 @@ public function __get( $key = '' ) {
 				'orderby'       => 'cart_index',
 				'order'         => 'ASC',
 				'no_found_rows' => true,
+				'number'        => 100,
```

Comment:
I agree on both of these suggestions.

1. Make the default 200 (it should have minimal impact on the query)
2. Make the `__get` method call the helper function instead of replicating things so we can don't have to update this same logic in the future.